javascript課題①-3.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+    const tasks = ["掃除","買い物","散歩"];
+
+    function tasklist() {
+    console.log("=========================");
+    console.log("現在持っているタスク一覧");
+    console.log("=========================");
+    tasks.forEach(function(item, index, array) {
+        console.log(String.raw`${index}:${item}`);
+    });
+  }
+
+    tasklist();
+    const addtask = prompt("タスクを追加してください");
```

Comment:
```suggestion
    const addTask = prompt("タスクを追加してください");
```

変数の命名は `camelCase` で行いましょう。
https://qiita.com/codeship_tech/items/22a657a758af5f5ef067